infrastructure/bicep/parameters/dev.json

```diff
@@ -228,7 +228,7 @@
       "value": "https://eth-mainnet.g.alchemy.com/v2"
     },
     "zchfGraphUrl": {
-      "value": "https://api.thegraph.com/subgraphs/name/frankencoin-zchf/frankencoin-subgraph"
+      "value": "https:/maindevponder.frankencoin.com"
```

Comment:
Missing `/`

Reply:
Very interessing - just copied it from my .env setting and in my .env setting it is with two slashes!?